pytorch_blade/src/compiler/mlir/converters/mhlo_conversion.cpp

```diff
@@ -199,15 +319,103 @@ class ConvertToMhloImpl {
   std::atomic_flag converted_ = ATOMIC_FLAG_INIT;
 };
 
+std::tuple<std::string, std::string, std::string, std::string>
+ConvertTorchToMhlo(std::shared_ptr<torch::jit::Graph> graph) {
+  std::shared_ptr<const torch::jit::Graph> const_graph = graph;
+  mlir::DialectRegistry registry;
+  RegisterDialects(registry);
+  ::mlir::torch::registerAllDialects(registry);
+  mlir::MLIRContext mlir_context(registry);
+  mlir_context.loadAllAvailableDialects();
+  bool enable_printing =
+      env::ReadBoolFromEnvVar("TORCH_BLADE_DEBUG_LOG", false);
+  if (enable_printing) {
```

Comment:
Do you want to use an explicit env flag to enable multithreading compilation?

Reply:
No, we disable multithreading compilation because MLIR printing is only capable under single-thread.